test/Elastic.Apm.Tests/StackTraceTests.cs

```diff
@@ -27,6 +28,9 @@ public class StackTraceTests
 		[Fact]
 		public async Task HttpClientStackTrace()
 		{
+			if(RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+				return; //in CI on Windows line numbers don't seem to be available.
```

Comment:
Are you sure? Based on https://apm-ci.elastic.co/blue/organizations/jenkins/apm-agent-dotnet%2Fapm-agent-dotnet-mbp/detail/PR-374/3/tests/

> Existing failures - 1
> Initializing / Parallel / Linux / Test / HttpClientStackTrace – Elastic.Apm.Tests.StackTraceTests

Tests fail only on Linux.

Reply:
Yeah, they fail both on Linux and Windows. It was just not clear to me previously that they also fail on Linux - somehow Jenkins showed those differently. Anyway, in https://github.com/elastic/apm-agent-dotnet/pull/374/commits/69d196ebee5cffe6d1b0e1fd8ea8639383251671 I tried to disable it for windows - with that it's now clear that it fails everywhere.

Reply:
> > Fighting with CI - it seems that in CI we don't get line number info and from that reason we have 1 failing test.
> > Can't repro locally - trying to figure this out - I don't see yet how this PR changes anything on that part 🤷‍♂️
> 
> I see that the test fails only on Linux - is it possible that on Linux assemblies don't have debugging info with file names and line numbers? It's not clear why this started to happen with this PR but maybe CI machines when through some changes? What would be the simplest way to re-run tests on the current master on CI machines?

I merged something today, and with that all tests were green: https://github.com/elastic/apm-agent-dotnet/commit/0b66a3212e9df7b01c29752469cdb4b1b8efb45d (only thing made it red is that the test coverage is below target, but that's a known bug and the next build will make it good again. - I think there is no change between this branch and master in terms of jenkins and CI related things.